src/components/InfoTip/InfoTip.test.js

```diff
@@ -1,229 +1,119 @@
 import React from 'react';
-import { mount } from 'enzyme';
-import { InfoTip } from './index';
-import { ListGroup, ListGroupItem } from '../ListGroup';
-
-const testInfoTip = () => (
-  <InfoTip id="infotip-widget">
-    <InfoTip.Toggle>Messages: 2</InfoTip.Toggle>
-    <InfoTip.Menu>
-      <ListGroup>
-        <ListGroupItem>
-          <InfoTip.MenuItemIcon type="pf" name="info" /> Added Datasources
-          TestDS
-        </ListGroupItem>
-        <ListGroupItem>
-          <InfoTip.MenuItemIcon type="pf" name="info" /> Modified Datasources
-          ExampleDS
-        </ListGroupItem>
-      </ListGroup>
-      <InfoTip.MenuFooter>
-        <a href="#">Clear Messages</a>
-      </InfoTip.MenuFooter>
-    </InfoTip.Menu>
-  </InfoTip>
-);
-
-test('InfoTip renders properly with item children', () => {
-  const component = mount(
-    <InfoTip id="infotip-widget">
-      <InfoTip.Toggle>Messages: 2</InfoTip.Toggle>
-      <InfoTip.Menu>
-        <ListGroup>
-          <ListGroupItem>
-            <InfoTip.MenuItemIcon type="pf" name="info" /> Added Datasources
-            TestDS
-          </ListGroupItem>
-          <ListGroupItem>
-            <InfoTip.MenuItemIcon type="pf" name="info" /> Modified Datasources
-            ExampleDS
-          </ListGroupItem>
-        </ListGroup>
-        <InfoTip.MenuFooter>
-          <a href="#">Clear Messages</a>
-        </InfoTip.MenuFooter>
-      </InfoTip.Menu>
-    </InfoTip>
-  );
-
-  expect(component.render()).toMatchSnapshot();
+import { shallow } from 'enzyme';
+import InfoTip from './InfoTip';
+import { KEY_CODES } from '../../common/helpers';
+import { Dropdown } from '../Dropdown';
+
+const props = {
+  id: 'InfoTip',
+  children: 'children'
+};
+
+test('renders Dropdown', () => {
+  const view = shallow(<InfoTip {...props} />);
+  expect(view).toMatchSnapshot();
```

Comment:
same question - do you want to name these or just leave the name argument out ;) ?